docs/sources/open-source.md

```diff
@@ -8,42 +8,64 @@ title: Open Source
+
+For more information on production environment installation, refer to the following OSS Grafana OnCall [production environment helm chart](https://github.com/grafana/oncall/helm)
+
+
+## Slack integration for Grafana OnCall OSS
 
-## Production Environment
+The Slack integration for Grafana OnCall leverages Slack API features to provide a customizable and useful integration.
 
-We prepared the helm chart for production environment: https://github.com/grafana/oncall/helm
+### Before you begin
 
-## Slack Setup
+Review the following requirements to configure Slack for Grafana OnCall OSS:
 
-Grafana OnCall Slack integration use a lot of Slack API features: 
-- Subscription on Slack events requires OnCall to be externally available and provide https endpoint. 
-- You will need to register new Slack App.
+- Ensure your Grafana OnCall environment is running.
+- Grafana OnCall must be externally available and provide HTTPS endpoint to establish subscription on Slack events.
+- Create a new Slack App from Grafana OnCall app manifest.
 
-1. Make sure your OnCall is up and running.
+### Configure Slack for Grafana OnCall OSS
+
+1. Ensure your Grafana OnCall environment is up and running.
+
+2. Grafana OnCall must be accessible through HTTPS. For development purposes, it's recommend to use [localtunnel](https://github.com/localtunnel/localtunnel). For production purposes, consider establishing a proper web server with HTTPS termination. 
+For localtunnel, refer to the following configuration:
 
-2. You need OnCall to be accessible through https. For development purposes we suggest using [localtunnel](https://github.com/localtunnel/localtunnel). For production purposes please consider setting up proper web server with HTTPS termination. For localtunnel: 
 ```bash
 # Choose the unique prefix instead of pretty-turkey-83
 # Localtunnel will generate an url, e.g. https://pretty-turkey-83.loca.lt
 # it is referred as <ONCALL_ENGINE_PUBLIC_URL> below
 lt --port 8080 -s pretty-turkey-83 --print-requests
 ```
 
-3. If you use localtunnel, open your external URL and click "Continue" to allow requests to bypass the warning page.
+3. If using localtunnel, open your external URL and click **Continue** to allow requests to bypass the warning page.
 
 4. [Create a Slack Workspace](https://slack.com/create) for development, or use your company workspace.
 
-5. Go to https://api.slack.com/apps and click Create New App button
+5. Go to https://api.slack.com/apps and click **Create New App** 
 
-6. Select `From an app manifest` option and choose the right workspace
+6. Select `From an app manifest` option and select your workspace.
 
-7. Copy and paste the following block with the correct <YOUR_BOT_NAME> and <ONCALL_ENGINE_PUBLIC_URL> fields
+7. Copy and paste the following block. Be sure to replace `YOUR_BOT_NAME` and `ONCALL_ENGINE_PUBLIC_URL` fields with the appropriate information. 
```

Comment:
```suggestion
1. Replace the text with the following YAML code block . Be sure to replace `<YOUR_BOT_NAME>` and `<ONCALL_ENGINE_PUBLIC_URL>` fields with the appropriate information. 
```

Reply:
The original formatting for the user variables is both a best practice, and the actual format in the YAML code block.

Reply:
I'm going through this actual process as I review these steps